vars/withFortifySecrets.groovy

```diff
@@ -0,0 +1,15 @@
+def call(String product, String fortifyVaultName, Closure block) {
+  def fortifySecrets = [
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'fortify-on-demand-username', version: '', envVariable: 'FORTIFY_USER_NAME'],
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'fortify-on-demand-password', version: '', envVariable: 'FORTIFY_PASSWORD'],
+  ]
+
+  String theKeyVaultUrl = fortifyVaultName ? "https://${fortifyVaultName}.vault.azure.net" : "https://${product}-aat.vault.azure.net/"
```

Comment:
needs to use the environment.nonProdName field, can't assume it will be called `aat`

Reply:
done